tests/src/infinite-loop.rs

```diff
@@ -0,0 +1,10 @@
+//@ [coq,fstar] skip
+//@ [coq,fstar] subdir=misc
+//^note: this fails on the fstar and coq backends
+fn bar() {}
+
+fn foo() {
+    loop {
+        bar()
+    }
+}
```

Comment:
Note that the infinite loop test fails on two backends, and the match test fails on coq. Should I open issues for that?

Reply:
Where does it come from originally?
If it fails it means there are missing fails, for instance for the decrease clauses. I can look into it.